src/Provider/AbstractProvider.php

```diff
@@ -31,7 +31,9 @@
 
     public $responseType = 'json';
 
-    public $headers = null;
+    public $headers = [];
+
+    public $authorizationHeader;
```

Comment:
For the default providers that ship with this package? No impact, everything should be complete transparent.

For a custom provider, everything should be smooth, unless `fetchProviderData` has been heavily modified and the code somehow became incompatible. I tried my best to make sure that existing code would not break with these changes applied... I don't think any of the abstract changes will break existing functionality.